bin/server_stop.sh

```diff
@@ -18,15 +18,17 @@ else
 fi
 
 pidFilePath=$appdir/$PIDFILE
+PID="$(cat "$pidFilePath")"
 
 if [ ! -f "$pidFilePath" ] || ! kill -0 "$(cat "$pidFilePath")"; then
    echo 'Job server not running'
```

Comment:
I am not quite sure cause I do not think removal of PID is guaranteed after the Line 27 execusion